src/sentry/static/sentry/app/views/settings/account/accountSecurity/index.jsx

```diff
@@ -45,25 +48,23 @@ class AccountSecurity extends AsyncView {
           .requestPromise(`${ENDPOINT}${auth.authId}/`, {
             method: 'DELETE',
           })
-          .then(this.remountComponent, () => {
+          .then(this.props.remountParent, () => {
```

Comment:
Instead of doing thing, I think we'd want to basically move `handleDisable` all the way up to `Wrapper`